### PR TITLE
Re-declare necessary Perl dependencies for running tap tests in ./configure

### PR DESCRIPTION
**Pull request by berlin-ab**
This reverts commit 848733b6edb113b1c5c5cd2b8f039bbf439a6994.

Ran a dev pipeline to tests that compilation steps work, which they do.

### CODE REVIEW

**Comment by berlin-ab** [comment]
The build image now includes the necessary Perl dependencies.

**Comment by hlinnaka** [comment]
Can you rephrase the commit message to remove the double-negative, "revert revert", please?

**Comment by berlin-ab** [comment]
> Can you rephrase the commit message to remove the double-negative, "revert revert", please?

Done. Thanks for the recommendation.
